src/lakefs_spec/client_helpers.py

```diff
@@ -26,6 +35,25 @@ def commit(
     message: str,
     metadata: dict[str, str] | None = None,
 ) -> Commit:
+    """Creates a new commit of all uncommitted changes on the branch in the lakeFS file storage
```

Comment:
Please end with a full stop (".").

Also, do we break the line after the multiline string """? I always like to do this since it's easier to read.